2_segmenting_data/Segmentation_Pipeline.ipynb

```diff
@@ -178,7 +178,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.8.13 ('2.segment-NF1-data')",
+   "display_name": "Python 3.8.13 ('download-NF1-data')",
```

Comment:
Noticing this change for the segmentation work, it looks like the kernel may have been changed to the download kernel. Is this accurate for the intention of this notebook?

Reply:
Huuuh

@d33bs Can you help me next week with this? I have no idea what this is or how this happened.

Reply:
@jenna-tomkinson - sure thing! I believe this sometimes can happen when a notebook is loaded within another environment.

Reply:
@d33bs I think I just fixed it!